www/js/tablet.js

```diff
@@ -1299,6 +1299,40 @@ function saveConfigValues(){
   hideModal('configuration-popup');
 }
 
+function togglePlay(play) {
+  document.getElementById('playButtonDiv').style.display = play ? 'block' : 'none';
+  document.getElementById('pauseButtonDiv').style.display = play ? 'none' : 'block';
+}
+
+function togglePlayPause(fromPlay) {
+  if (fromPlay) {
+    if (STATE !== 'Idle' && STATE !== 'Hold') {
```

Comment:
What happens in the hold state? Do the motors continue to hold position? We might want to test to make sure that is safe for them to do because I think maybe they can draw a lot of current without rotating enough to cool themselves that way

Reply:
I think so? I know that the pauseGcode "!" call puts the machine in this state. From what I see, the machine is frozen. I think the idea of "pause" is that you'd want to continue soon, but I can run this longer and see what it does.

Reply:
I'm a little worried about folks pausing their cut to come back the next day or something